packages/fast-components-class-name-contracts-msft/src/progress/index.ts

```diff
@@ -4,16 +4,51 @@ import { ProgressClassNameContract as BaseProgressClassNameContract } from "@mic
  * The class name contract for the progress component
  */
 export interface ProgressClassNameContract extends BaseProgressClassNameContract {
+    /**
+     * The root of the circular progress component
+     */
+    progressCircular?: string;
+
+    /**
+     * The progress size control modifier
+     */

```

Comment:
I'm not following what these three classes are for by these descriptions - can you elaborate and align with the naming conventions I mentioned?

Reply:
I'm also curious here, if `progress` is the root element and these are all modifiers on the root element I would expect `progress__circular`, `progress__control`, `progress__container`, `progress__page`. Your styling would then presumably be:

```
{
    progress__circular: {
        "&$progress__control": {
            // more specific stylings for circular control
        }
    }
}
```

Reply:
the size modifiers only apply to the circular progress.